core/src/runtime/RuntimeResolver.ts

```
@@ -269,6 +269,19 @@ export default class RuntimeResolver {
     }
 
+    @Query()
+    runtimeGetNetworkMetrics(): String {
+        return JSON.stringify({
+            "metrics": {},
+            "stats": {}
+        })
+    }
+
+    @Mutation(returns => Boolean)
+    runtimeRestartHolochain(): Boolean {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)